data/World/Overworld.json

```diff
@@ -649,6 +741,9 @@
         "font_color": "Yellow",
         "scene": "Haunted Wasteland",
         "hint": "the Haunted Wasteland",
+        "locations": {
+            "Wasteland Crate Near Colossus": "True"
+        },
```

Comment:
I suggest a new region for this crate in the middle of wasteland crossing. You have to at least partially reverse wasteland to reach the crate coming from colossus side, which I don't think should be in default logic. We have a reverse wasteland trick that should be applying but isn't.

Reply:
So this crate can be reached from colossus by walking directly straight from the entrance. If it was any more difficult than that, I think I would agree it should be some kind of trick. But it's really easy to get.

Reply:
I would probably agree with you if you could see the crate from the entrance. But you do have to know the general direction to walk. I'm not okay with any amount of walking blindly into the sandstorm in default logic.